packages/bpk-component-calendar/src/BpkCalendarContainer-test.tsx

```diff
@@ -16,18 +16,14 @@
  * limitations under the License.
  */
 
-import { render, screen, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import { weekDays, formatDateFull, formatMonth } from '../test-utils';
 
 import BpkCalendarContainer from './BpkCalendarContainer';
 import { CALENDAR_SELECTION_TYPE } from './custom-proptypes';
 
-const createNodeMock = () => ({
```

Comment:
this was left from some old tests I believe as the way that the tests are currently structured, this doesn't do anything and is not supported by RTL